queue/driver_test.go

```diff
@@ -473,6 +473,191 @@ func (s *DriverSuite) TestJobsMethodReturnsAllJobs() {
 	s.Equal(counter, len(mocks))
 }
 
+func (s *DriverSuite) TestRetryableJobsReturnsAllRetryableJobs() {
```

Comment:
Do we need to test with groups?

Reply:
I changed the driver suite to run all the tests with and without groups to ensure they work the same way.